api/ticket/Ticket.js

```diff
@@ -6,12 +6,19 @@ const ticketSchema = new Schema({
   screening: {
     type: Schema.Types.ObjectId,
     ref: 'Screening',
+    required: true,
   },
   row: Number,
   column: Number,
```

Comment:
row and column are in Seat, so You can remove these keys, probably.

Reply:
Done